### PR TITLE
UI for return user flow to view list of projects and create a new one

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

File: client/src/event/Auth.vue [hunk 1]
```diff
@@ -52,6 +56,11 @@
   export default {
     name: 'Auth',
     mixins: [FetchEventMixin],
+    props: {
+      blarp: {
```
Comment (Wardormeur) [inline client/src/event/Auth.vue]: blarp? :>

File: client/src/project/Form.vue [hunk 2]
```diff
@@ -401,6 +405,7 @@
           query: {
             verified: 1,
             deleted: 0,
+            stage: { ne$: 4 },
```
Comment (Wardormeur) [inline client/src/project/Form.vue]: ahahaha :')

File: client/src/project/Form.vue [hunk 3]
```diff
@@ -411,13 +416,73 @@
...
+            eventLabel: this.event.id,
+          });
+          return project;
+        } catch (err) {
+          this.error = err;
+        }
+      },
+      onBeforeUnload(e) {
+        e.returnValue = 'Are you sure you don\'t want to complete your registration application?';
+        return 'Are you sure you don\'t want to complete your registration application?';
```
Comment (Wardormeur) [inline client/src/project/Form.vue]: Not sure to understand why the string is set twice here, but eh

File: client/src/event/Auth.vue [hunk 4]
```diff
@@ -1,47 +1,51 @@
...
+        </div>
+      </label>
+      <div class="row row-double-margin">
+        <div class="col text-center">
+          <button type="submit" class="btn btn-primary" :class="{ disabled: !approval }">Next step – project details</button>
+        </div>
       </div>
+      <div v-if="error" class="row error-message">
+        <div class="col">
+          <p v-if="error.status === 409">
```
Comment (Wardormeur) [inline client/src/event/Auth.vue]: Double handling of 409 ?

File: client/src/event/Auth.vue [hunk 5]
```diff
@@ -1,47 +1,51 @@
...
-    <div class="row">
-      <div class="col">
-        <label>This form is to register a project for Coolest Projects International 2018 which will be held on Saturday 26th May 2018 in RDS Simmonscourt, Dublin, Ireland. Just one person should register for each project.</label>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col">
-        <input type="email" v-model="email" class="full-width-block" placeholder="you@email.com" required />
-      </div>
+    <div v-if="error && error.status === 409">
```
Comment (Wardormeur) [inline client/src/event/Auth.vue]: Handling of 409

File: client/src/event/Auth.vue (hunk 1 above)
Comment (DanielBrierton) [reply client/src/event/Auth.vue]: Whoops, was testing something... 😅

File: client/src/project/Form.vue (hunk 2 above)
Comment (DanielBrierton) [reply client/src/project/Form.vue]: 😂 😂 😂 😂 😂

File: client/src/project/Form.vue (hunk 3 above)
Comment (DanielBrierton) [reply client/src/project/Form.vue]: Yeah, I could make a const and assign I guess. Would make more sense. I think this mostly came from "huh, return alone doesn't work... \*copy\* \*paste\*. Cool, it works now!"

Comment (Wardormeur) [reply client/src/project/Form.vue]: That's fine, no biggie, just not sure "how" is e.returnValue used differently from the return itself

Comment (DanielBrierton) [reply client/src/project/Form.vue]: Because web standards https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload

😸

Comment (Wardormeur) [reply client/src/project/Form.vue]: Am i allowed to say it's a crappy standard? :D

**Comment by DanielBrierton** [comment]
Closing in favour of #16 and #17 which cover these commits have been cherry-pick'd to.

**Pull request closed without merging**